### DISCUSSION TITLE
luna wormhole not received from sollet wallet to terra

### DISCUSSION THREAD

**Discussion by lunatech2022** [wormhole-foundation/wormhole#719] [Problems / bugs] [answered]
Can someone please help?

21ThxUSc3GPZLFJRuLRo9x8RSsFhMMUSwBmW3wHqGERCFBokZPkpuRyakwsqp1MWX9b4W4od4nZRwe8K5XLEAitU

I tried to use the wormhole to transfer my luna from my Sollet wallet to my Terra Bridge wallet. The transaction was getting rejected when I hit redeem due to an error 

"ouluna is smaller than 35204uluna insufficient funds"

I thought the transaction failed and closed the tab

now I do not see the Luna in my sollet wallet and it was never successfully sent to my luna wallet. 

Can anyone please help on what I can do push the transaction through or cancel????

Thank you.

**Comment by evan-gray** [comment, marked as answer]
Since this post, this transaction has been redeemed.
